antenna/app.py

```diff
@@ -37,7 +85,7 @@ class BreakpadSubmitterResource(RequiredConfigMixin):
 
     required_config.add_option(
         'crashstorage_class',
-        default='antenna.external.boto.crashstorage.BotoS3CrashStorage',
+        default='antenna.external.crashstorage_base.NoOpCrashStorage',
```

Comment:
Figured I'd make the default be `NoOpCrashStroage`. In Socorro-land, we like to make the default whatever production uses. I see the usefulness of that, but in Socorro-land, we have no ability to run a prod-like setup whereas with Antenna, we'll have a dockerized prod-like setup. So I claim I'm solving the underlying need, but in a different way.

We can redecide that later if it proves to be an issue.